kong/tracing/propagation.lua

```diff
@@ -1,11 +1,13 @@
 local to_hex = require "resty.string".to_hex
 local table_merge = require "kong.tools.utils".table_merge
+local otlp = require "kong.plugins.opentelemetry.otlp"
```

Comment:
referring to the code of a plugin in the general module looks a bit strange.

Reply:
To be honest, I prefer a redesign of the whole structure.

Reply:
move `translate_span` func to propagation?

Reply:
Please referer to KAG-876.

Reply:
Basically, I feel the detail of every format should be handled by each plugin (by registering handles to parse headers and emit headers); and the process of propagation should be driven by the tracing PDK.